diff --git a/configure b/configure
--- a/configure
+++ b/configure
@@ -65 +65 @@
-pkg_config="pkg-config"
+pkg_config="${PKG_CONFIG:-pkg-config}"
@@ -905 +905 @@
-    if test "$cross_prefix" = "" -a "$pkg_config" != "no"; then
+    if test "$pkg_config" != "no"; then
@@ -1431 +1431 @@
-        if test "$cross_prefix" = "" -a "$is_port" != "yes" -a "$pkg_config" != "no" -a "$pkg_cfg_name" != ""; then
+        if test "$is_port" != "yes" -a "$pkg_config" != "no" -a "$pkg_cfg_name" != ""; then
