diff --git a/configure b/configure
--- a/configure
+++ b/configure
@@ -71 +71 @@
-pkg_config="pkg-config"
+pkg_config="${PKG_CONFIG:-pkg-config}"
@@ -902,7 +902,7 @@
 fi

 #check pkg_config
-if test "$cross_prefix" = "" -a "$pkg_config" != "no"; then
+if test "$pkg_config" != "no"; then
     if ! $pkg_config --version >/dev/null 2>>$logs ; then
         pkg_config="no"
     fi
@@ -1429,6 +1429,6 @@
     #test pkgconfig
     if test "$forced" != "local" -a "$custom" != "yes" ; then
-        if test "$cross_prefix" = "" -a "$is_port" != "yes" -a "$pkg_config" != "no" -a "$pkg_cfg_name" != ""; then
+        if test "$is_port" != "yes" -a "$pkg_config" != "no" -a "$pkg_cfg_name" != ""; then
             if $pkg_config --exists $pkg_cfg_name ; then
                 pck_cflags=`$pkg_config --cflags $pkg_cfg_name`
                 pck_ldflags=`$pkg_config --libs $pkg_cfg_name`
